src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.mocks.ts

```diff
@@ -3,7 +3,7 @@ import {
     mockTab2,
     mockMostCommented,
     mockMostShared,
-} from '../MostViewed.mocks';
+} from '@root/fixtures/mostViewed';
```

Comment:
Nice. Pulling this in here is much more efficient than restructuring everything 👍